### PR TITLE
REP-93 regions

### PR DESCRIPTION
**Pull request by edwh**
This adds a very basic support for regions on the client side only.

- A region has a bounding box, and also a tagline.
- We show a map view which fits in that bounding box (and therefore should adapt to screen size).
- The map contains only the businesses inside that region. We don't have a UK-wide region at the moment.
- The link shared contains the region.

You can test this using these URLs.

http://localhost:3000/ 
http://localhost:3000/?region=Wales
http://localhost:3000/?region=London

This is a quick approach which may do for Wales - see what you think.

### CODE REVIEW

File: components/Map.vue [hunk 1]
```diff
@@ -96,11 +124,9 @@ export default {
           )
         })
 
-        if (businesses.length) {
```
Comment (ngm) [inline components/Map.vue:99]: How come this check no longer needed?

File: mixins/global.js [hunk 2]
```diff
@@ -1,7 +1,25 @@
 // Global mixin so that every component can access the logged in state and user.
 import Vue from 'vue'
+import { REGION_WALES, TAGLINE_LONDON, TAGLINE_WALES } from '@/regions'
```
Comment (ngm) [inline mixins/global.js:3]: What's the @ here?  Don't think I've seen that before.

File: regions.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+export const REGION_LONDON = 'London'
```
Comment (ngm) [inline regions.js:1]: Makes perfect sense here for now, just pondering out loud if you think one day these regions and corresponding info might come from the DB.

**Review by ngm — APPROVED** [review]
Left a couple of comments in the Jira story too.

File: components/Map.vue (hunk 1 above)
Comment (edwh) [reply components/Map.vue:99]: This might be resolved by merging, I think.  The check is there now.

File: mixins/global.js (hunk 2 above)
Comment (edwh) [reply mixins/global.js:3]: It's like ~ but avoids bothering with the extension.  See https://stackoverflow.com/a/42753045/775258

File: regions.js (hunk 3 above)
Comment (edwh) [reply regions.js:1]: Yes, definitely.

**Pull request merged**